internal/isoeditor/rhcos.go

```diff
@@ -0,0 +1,146 @@
+package isoeditor
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"regexp"
+
+	"github.com/openshift/assisted-service/internal/isoutil"
+	"github.com/openshift/assisted-service/restapi/operations/bootfiles"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	// BaseIsoTempDir is a temporary directory pattern for the extracted base ISO
+	BaseIsoTempDir string = "baseiso"
+)
+
+type Editor interface {
+	CreateMinimalISOTemplate(serviceBaseURL string) (string, error)
+	AddRootFSURL(serviceBaseURL string) error
```

Comment:
Also, this feels like it shouldn't really be in the public API. We should be able to call this directly from the test as a private method if that's the only reason we need it.

Reply:
done